cmd/jaeger/integration/receivers/storagereceiver/.nocover

```diff
@@ -0,0 +1 @@
+FIXME
```

Comment:
may want to pull storagereceiver into a separate PR as well, and add the tests to it (most other v2 packages already have tests)